public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -0,0 +1,823 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect, Fragment } from 'react';
+import {
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiTitle,
+  EuiButton,
+  EuiFormFieldset,
+  EuiCheckableCard,
+  EuiSpacer,
+  EuiIcon,
+  EuiText,
+  EuiSwitch,
+  EuiButtonIcon,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCheckbox,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiFieldNumber,
+  EuiCallOut,
+  EuiButtonEmpty,
+  EuiPanel,
+} from '@elastic/eui';
+import './styles.scss';
+import {
+  createAugmentVisSavedObject,
+  ISavedAugmentVis,
+  VisLayerExpressionFn,
+} from '../../../../../../src/plugins/vis_augmenter/public';
+import { useDispatch } from 'react-redux';
+import { snakeCase, find, isEmpty, get } from 'lodash';
+import {
+  Field,
+  FieldArray,
+  FieldArrayRenderProps,
+  FieldProps,
+  Form,
+  Formik,
+  FormikHelpers,
+} from 'formik';
+import {
+  createDetector,
+  startDetector,
+} from '../../../../public/redux/reducers/ad';
+import { EmbeddablePanel } from '../../../../../../src/plugins/embeddable/public';
+import './styles.scss';
+import EnhancedAccordion from '../EnhancedAccordion';
+import MinimalAccordion from '../MinimalAccordion';
+import {
+  FEATURE_TYPE,
+  UNITS,
+  Detector,
+} from '../../../../public/models/interfaces';
+import { DataFilterList } from '../../../../public/pages/DefineDetector/components/DataFilterList/DataFilterList';
+import {
+  convertTimestampToNumber,
+  getError,
+  isInvalid,
+  validateDetectorName,
+  validateFeatureName,
+  validatePositiveInteger,
+} from '../../../../public/utils/utils';
+import { CUSTOM_AD_RESULT_INDEX_PREFIX } from '../../../../server/utils/constants';
+import {
+  focusOnFirstWrongFeature,
+  initialFeatureValue,
+} from '../../../../public/pages/ConfigureModel/utils/helpers';
+import { FeaturesFormikValues } from '../../../../public/pages/ConfigureModel/models/interfaces';
+import {
+  getIndices,
+  getMappings,
+} from '../../../../public/redux/reducers/opensearch';
+import {
+  featuresToUIMetadata,
+  formikToFeatureAttributes,
+  formikToFilterQuery,
+} from '../../../../public/pages/ReviewAndCreate/utils/helpers';
+import { FormattedFormRow } from '../../../../public/components/FormattedFormRow/FormattedFormRow';
+import { FeatureAccordion } from '../../../../public/pages/ConfigureModel/components/FeatureAccordion';
+import {
+  AD_DOCS_LINK,
+  MAX_FEATURE_NUM,
+} from '../../../../public/utils/constants';
+
+function AddAnomalyDetector({
+  embeddable,
+  closeFlyout,
+  notification,
+  services,
```

Comment:
We should use `getNotifications()` from `services` file instead of passing down`notification` as a prop.

And i think `services` can be removed, I don't see it used in the file